tests/bluetooth/audio/ascs/src/test_ase_state_transition.c

```diff
@@ -25,7 +25,10 @@
 
 #include "test_common.h"
 
-struct test_sink_ase_state_transition_fixture {
+#define test_sink_ase_state_transition_fixture test_ase_state_transition_fixture
+#define test_source_ase_state_transition_fixture test_ase_state_transition_fixture
```

Comment:
What is the purpose of these?

Reply:
Test suite's fixture has to be named after test suite followed by `_fixture` suffix.
Example: `my_test_suite` `my_test_suite_fixture`.
In this case I have 2 test suits: `test_sink_ase_state_transition` and `test_source_ase_state_transition`, that share common `struct test_ase_state_transition_fixture`. Thus I added those defines to make it compile :D

Reply:
> Test suite's fixture has to be named after test suite followed by _fixture suffix.

Is that something that the framework requires? 

But yeah I guess this makes sense if that is required, and it's more maintainable than defining two identical structs with different names :)